apps/studio/components/interfaces/Organization/GeneralSettings/OptInToOpenAIToggle.tsx

```diff
@@ -1,71 +1,65 @@
 import Link from 'next/link'
-import { cn, Collapsible_Shadcn_, CollapsibleTrigger_Shadcn_, CollapsibleContent_Shadcn_ } from 'ui'
-import { useState } from 'react'
-import { ChevronRight } from 'lucide-react'
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogSection,
+  DialogTitle,
+  DialogTrigger,
+  Button,
+} from 'ui'
 
 interface OptInToOpenAIToggleProps {
   className?: string
 }
 export default function OptInToOpenAIToggle({ className }: OptInToOpenAIToggleProps) {
-  const [open, setOpen] = useState(false)
-
   return (
-    <Collapsible_Shadcn_ open={open} onOpenChange={setOpen} className={cn('mt-4', className)}>
-      <CollapsibleTrigger_Shadcn_ asChild>
-        <div className="flex items-center space-x-2 cursor-pointer">
-          <ChevronRight
-            strokeWidth={2}
-            size={16}
-            className={cn('transition-all', open ? 'rotate-90' : '')}
-          />
-          <p className="text-sm text-foreground-light underline">
-            Important information regarding opting in
-          </p>
-        </div>
-      </CollapsibleTrigger_Shadcn_>
-      <CollapsibleContent_Shadcn_>
-        <div className="space-y-2 py-4 text-sm text-foreground-light">
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button
+          type="text"
+          className="text-brand border-b p-0 hover:bg-transparent hover:text-foreground text-sm"
+        >
+          Learn more about data privacy
```

Comment:
just feedback but i found it weird that the text is green, but hover makes it white hahaha 😅